core/src/main/java/org/jboss/set/mjolnir/archive/ldap/LdapScanningBean.java

```diff
@@ -203,4 +195,38 @@ private void createUniqueUserRemoval(Set<String> existingUserNamesToProcess, Str
             em.persist(removal);
         }
     }
+
+    private List<RegisteredUser> getRegisteredUser() {
+        List<RegisteredUser> listOfUsersTable = em.createNamedQuery(RegisteredUser.FIND_ALL).getResultList();
+        return listOfUsersTable;
+    }
+
+    public List<RegisteredUser> getInvalidLdapUser() throws IOException, NamingException {
```

Comment:
On the other hand I think if this should rather be a private (or package private, if we need to call it from a test case) method of the `InvalidResponsiblePersonsTable` (see bellow).

It's unlikely we would need to use it from elsewhere...